.github/CONTRIBUTING.md

```diff
@@ -30,7 +32,13 @@ $ git push origin master --force
 
 ## Issues
 
-Check the [Ideas](/doc/IDEAS.md) page if you are searching for a good topic to start with. You can also visit the [issue-tracker](https://github.com/ElektraInitiative/libelektra/issues) and filter the list by pre-defined labels like [good first issue](https://github.com/ElektraInitiative/libelektra/labels/good%20first%20issue).Do not hesitate to open a new issue if you want to ask a question, report or fix a bug or approach new topics.
+Check the [Ideas](/doc/IDEAS.md) page if you are searching for a good topic to start with. You can also visit the [issue-tracker](https://github.com/ElektraInitiative/libelektra/issues) and filter the list by pre-defined labels like [good first issue](https://github.com/ElektraInitiative/libelektra/labels/good%20first%20issue). Do not hesitate to open a new issue if you want to ask a question, report or fix a bug or approach new topics.
+
+## Ways to Contribute
```

Comment:
The section above also already has one "way to contribute" (i.e. looking for issues in the issue tracker).

Perhaps this should be "More ways to contribute"?